include/mamba/core/context.hpp

```diff
@@ -134,6 +134,7 @@ namespace mamba
 
         VerificationLevel safety_checks = VerificationLevel::kWarn;
         bool extra_safety_checks = false;
+        bool artifact_verif = false;
```

Comment:
how about `verify_artifacts`?
It's almost the same length and proper words. Would also be good for yaml option or flag

`verify_artifacts: true`

`--verify-artifacts`

Reply:
done, this is RC configurable using `verify_artifacts` and also environment variable configurable using `MAMBA_VERIFY_ARTIFACTS`